ci/scripts/run_ci_local.sh

```diff
@@ -59,9 +59,12 @@ GIT_COMMIT=$(git log -n 1 --pretty=format:%H)
 
 LOCAL_CI_TMP=${LOCAL_CI_TMP:-${MORPHEUS_ROOT}/.tmp/local_ci_tmp}
 CONTAINER_VER=${CONTAINER_VER:-241004}
-CUDA_VER=${CUDA_VER:-12.1}
+CUDA_VER=${CUDA_VER:-12.5}
+CUDA_FULL_VER=12.5.1
```

Comment:
If we allow users to override `CUDA_VER` but not `CUDA_FULL_VER` this could potentially end up with an inconsistent situation. 
```suggestion
CUDA_VER=${CUDA_VER:-12.5}
CUDA_FULL_VER=${CUDA_FULL_VER:-12.5.1}
```